recipes/ocgis/run_test.py

```diff
@@ -1,3 +1,13 @@
+import os
 from ocgis.test import run_simple
 
+# Set GDAL_DATA. This is done normally done by the activate script,
+# but this doesn't happen in the testing environment.
+if 'LIBRARY_PREFIX' in os.environ:  # Windows.
+    gdalData = os.path.join(os.environ['LIBRARY_PREFIX'], 'share', 'gdal')
+else:  # Linux/OS X.
+    gdalData = os.path.join(os.environ['PREFIX'], 'share', 'gdal')
```

Comment:
@ocefpaf, this should be good to go with the ``v1.3.1`` tag!

@pelson, I'm not enamored with the solution either. I'd love to find an alternative.